### PR TITLE
swift post

### PR DESCRIPTION
**Pull request by hannes**

### CODE REVIEW

File: _posts/2023-04-21-swift.md [hunk 1]
```diff
@@ -0,0 +1,190 @@
...
+
+Today we’re excited to announce the [DuckDB API for Swift](https://github.com/duckdb/duckdb-swift). It enables developers on Swift platforms to harness the full power of DuckDB using a native Swift interface with support for great Swift features such as strong typing and concurrency. The API is available not only on Apple platforms, but on Linux too, opening up new opportunities for the growing Swift on Server ecosystem.
+
+## What’s included
+DuckDB is designed to be fast, reliable and easy to use, and it’s this philosophy that also guided the creation of our new Swift API.
+
+This initial release supports many of the great features of DuckDB right out of the box, including:
+
+- Queries via DuckDB’s enhanced SQL dialect: In addition to basic SQL, DuckDB supports arbitrary and nested correlated subqueries, window functions, collations, complex types (Swift arrays and structs), and more.
+- Import and export of JSON and CSVs: Beyond its built-in and super-efficient native file format, DuckDB supports reading in, and exporting out to, JSON and CSV files.
```
Comment (Mytherin) [inline _posts/2023-04-21-swift.md:19]: Maybe add Parquet?

**Review by Mytherin — COMMENTED** [review]
Looks great - one minor comment

File: _posts/2023-04-21-swift.md (hunk 1 above)
Comment (hannes) [reply _posts/2023-04-21-swift.md]: done thanks

**Comment by Alex-Monahan** [comment]
Looks great!

**Pull request merged**